docs/src/tutorials/getting-started-rstudio/README.md

```diff
@@ -0,0 +1,13 @@
+# My first RStudio Project
+
+This tutorial is recommended for all users who are starting a new project in RStudio for the
+first time.
+
+This tutorial will walk you through setting up a basic RStudio project which crucially uses
+[renv](https://rstudio.github.io/renv/articles/renv.html), a tool designed to capture all
+of your packages alongside the code and which is needed within DataLabs for your project to
+work consistently.
+
+1. [Create a project](create-project.md)
```

Comment:
In SUMMARY.md, this is 'Create a new RStudio Project', which is probably a better title.

Reply:
@iwalmsley , I think this still needs doing in README.md

Reply:
Ah yes! Have fixed this now thanks.